client/src/components/StepWizard/WizardCheckboxItem.js

```diff
@@ -19,19 +18,23 @@ export const WizardCheckboxWrapper = styled.div`
   }
 `;
 
-export const WizardCheckboxItem = styled(CheckboxItem)`
+const CheckboxItemStyles = styled.div`
   font-family: ${theme.typography.font.family.display};
   font-size: ${theme.typography.size.large};
   background-color: ${white};
+  padding: 2rem;
   margin-bottom: 1rem;
   width: 100%;
+  cursor: pointer;
 
   .am-list-line {
     &:after {
       background-color: unset !important;
     }
+  } 
+  .am-checkbox-input {
+    cursor: pointer; 
```

Comment:
added a pointer to the checkbox input and checkbox item div. Thanks @anniegiang!